app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -19,6 +19,6 @@ class VerifyCsrfToken extends Middleware
      * @var array
      */
     protected $except = [
-        //
+        'login/saml2/callback',
```

Comment:
Is it safe to add the `login` route there?

Reply:
Good catch, I have to change it for 'auth/saml2/callback'!
This is a glitch explained in the [documentation](https://socialiteproviders.com/Saml2/#saml-protocol). The SAML2 protocol wants to call the callback using a POST request, instead of a GET request like all other OAuth calls. But there is no CSRF token passed to this callback, and we have to exclude it here.